tests/test_benchmark_cli.py

```diff
@@ -135,39 +136,87 @@ def gather_all_options_in_list(table):
 
 
 def test_write_multi_duplicated_api(file=generate_test_path('integers.ion')):
     table = execution_with_command(['write', file, '--api', 'simple_ion', '--api', 'event', '--api', 'event'])
-    assert gather_all_options_in_list(table) == sorted([('event', 'ion_binary'), ('simple_ion', 'ion_binary')])
+    assert gather_all_options_in_list(table) == sorted([('event', 'ion_binary', 'file'), ('simple_ion', 'ion_binary', 'file')])
 
 
 def test_read_multi_format(file=generate_test_path('integers.ion')):
     table = execution_with_command(['read', file, '--format', 'ion_text', '--format', 'ion_binary'])
-    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_binary'), ('simple_ion', 'ion_text')])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_binary', 'file'), ('simple_ion', 'ion_text', 'file')])
 
 
 def test_write_multi_format(file=generate_test_path('integers.ion')):
     table = execution_with_command(['write', file, '--format', 'ion_text', '--format', 'ion_binary'])
-    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text'), ('simple_ion', 'ion_binary')])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text', 'file'), ('simple_ion', 'ion_binary', 'file')])
 
 
 def test_read_multi_duplicated_format(file=generate_test_path('integers.ion')):
     table = execution_with_command(['read', file, '--format', 'ion_text', '--format', 'ion_binary', '--format', 'ion_text'])
-    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text'), ('simple_ion', 'ion_binary')])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text', 'file'), ('simple_ion', 'ion_binary', 'file')])
 
 
 def test_write_multi_duplicated_format(file=generate_test_path('integers.ion')):
     table = execution_with_command(['write', file, '--format', 'ion_text', '--format', 'ion_binary', '--format', 'ion_text',])
-    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text'), ('simple_ion', 'ion_binary')])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', 'ion_text', 'file'), ('simple_ion', 'ion_binary', 'file')])
+
+
+@parametrize(
+    *tuple((f.value for f in Format.Format if Format.format_is_json(f.value)))
+)
+def test_write_json_format(f):
+    table = execution_with_command(['write', generate_test_path('integers.ion'), '--format', f'{f}'])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', f'{f}', 'file')])
+
+
+@parametrize(
+    *tuple((f.value for f in Format.Format if Format.format_is_json(f.value)))
+)
+def test_read_json_format(f):
+    table = execution_with_command(['read', generate_test_path('integers.ion'), '--format', f'{f}'])
+    assert gather_all_options_in_list(table) == sorted([('simple_ion', f'{f}', 'file')])
+
+
+@parametrize(
+    *tuple((f.value for f in Format.Format if Format.format_is_json(f.value)))
+)
+def test_write_json_format(f):
```

Comment:
Should we also include the test case for `test_write_cbor_format` here?

Reply:
[87e7f65](https://github.com/amazon-ion/ion-python/pull/243/commits/87e7f65f9d05fc527820226a5dc93c9ee1d22954) addressed them.